lib/stack_master/identity.rb

```diff
@@ -19,5 +24,17 @@ def region
     def sts
       @sts ||= Aws::STS::Client.new(region: region)
     end
+
+    def iam
+      @iam ||= Aws::IAM::Client.new(region: region)
+    end
+
+    def contains_account_id?(ids)
+      ids.include?(account)
+    end
+
+    def contains_account_alias?(aliases)
+      account_aliases.any? { |account_alias| aliases.include?(account_alias) }
```

Comment:
Perhaps this should only be checked (and the API call made) if the provided values look like aliases, ie. they're not numbers.

Reply:
I thought about this too, but the problem with this approach is that it only covers the case where the check is performed, not when the error message is being generated in the cli.rb file.

I think the way to cover all cases is to rescue the permissions error in the `account_aliases` method and hard fail with a useful error message.

Reply:
👍 Let's do that.